app/app.js

```diff
@@ -187,24 +190,28 @@ function processHeroProfile(){
       profileDescription.html('<p>'+heroDesc+'</p>');
       profileRealName.html(heroSecretId)
 
+      // Give the profile header the proper images
+      profileHeader.background = 'url(' + heroBackground + ')no-repeat center center / cover';
```

Comment:
That's what I plan too, but for now, I just try to get it as expected, but I agree :+1: